FieldOpt/Simulation/simulator_interfaces/driver_file_writers/driver_parts/ecl_driver_parts/welspecs.cpp

```diff
@@ -2,63 +2,65 @@
 #include <iostream>
 
 namespace Simulation {
-    namespace SimulatorInterfaces {
-        namespace DriverFileWriters {
-            namespace DriverParts {
-                namespace ECLDriverParts {
+namespace SimulatorInterfaces {
+namespace DriverFileWriters {
+namespace DriverParts {
+namespace ECLDriverParts {
 
-                    Welspecs::Welspecs(QList<Model::Wells::Well *> *wells)
-                    {
-                        initializeBaseEntryLine(10);
-                        head_ = "WELSPECS";
-                        foot_ = "/\n\n";
-                        for (int i = 0; i < wells->size(); ++i) {
-                            if (wells->at(i)->trajectory()->GetWellBlocks()->size() > 0)
-                                entries_.append(createWellEntry(wells->at(i)));
-                        }
-                    }
+Welspecs::Welspecs(QList<Model::Wells::Well *> *wells)
+{
+    initializeBaseEntryLine(10);
+    head_ = "RPTRST\n'BASIC=3'\n/\n\nWELSPECS";
```

Comment:
I don't agree with this change. Writing this output slows down the simulation, and is of no use to the optimization. It should not be forced in by FieldOpt. Better to add it to the deck elsewhere (in the parts not touched by FieldOpt), or just add it after the optimization is done.

Reply:
Really? Never noticed. Is it really that much compared to everything else? If this is in, then one is able to view the performance of the  candidate solutions during runtime. The keywords needs to be added after SCHEDULE. If you are hard set against it, just revert the change, and merge.